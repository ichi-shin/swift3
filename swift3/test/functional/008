#!/bin/bash

# PUT Bucket lifecycle

. ./common

S3USER=tester

_s3_put /bucket

echo '<LifecycleConfiguration>
<Rule>
   <ID>Archive and then delete rule</ID>
   <Prefix>prefix_</Prefix>
   <Status>Enabled</Status>
   <Expiration><Days>0</Days></Expiration>
</Rule>
</LifecycleConfiguration>' > $tmp.lifecycle.xml
MD5=$(_md5 $tmp.lifecycle.xml) _s3_put /bucket?lifecycle -T $tmp.lifecycle.xml

echo '<LifecycleConfiguration>
<Rule>
   <ID>Archive and then delete rule</ID>
   <Prefix>prefix2_</Prefix>
   <Status>Enabled</Status>
   <Expiration><Days>1</Days></Expiration>
</Rule>
</LifecycleConfiguration>' > $tmp.lifecycle2.xml
MD5=$(_md5 $tmp.lifecycle2.xml) _s3_put /bucket?lifecycle -T $tmp.lifecycle2.xml
