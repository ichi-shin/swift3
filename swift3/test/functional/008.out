QA output created by 008
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester --contentMd5 U2caJjlmWSTwONBlU/U/dA== -- -X PUT -T /TMP.lifecycle.xml http://SWIFT_HOST/bucket?lifecycle... 200
> s3curl --id tester --contentMd5 eDUIEpKeDtC0qLR7RP6kkQ== -- -X PUT -T /TMP.lifecycle2.xml http://SWIFT_HOST/bucket?lifecycle... 200
